packages/cilium/package.yaml

```diff
@@ -1,5 +1,5 @@
-url: https://helm.cilium.io/cilium-1.9.8.tgz
-packageVersion: 08
+url: https://helm.cilium.io/cilium-1.10.4.tgz
+packageVersion: 09
```

Comment:
If the base tarball changes, we can start from 01 again

Reply:
done